### PR TITLE
chore: Added language chooser

### PR DESCRIPTION
**Pull request by kj-crypto**

### CODE REVIEW

File: electrum/gui/qt/installwizard.py [hunk 1]
```diff
@@ -197,10 +187,41 @@ def __init__(self, config: 'SimpleConfig', app: QApplication, plugins: 'Plugins'
...
 
+    def _set_gui_text(self):
+        self.setWindowTitle('ELCASH Wallet  -  ' + _('Install Wizard'))
+        self.back_button.setText(_('Back') if self.can_go_back() else _('Cancel'))
+        self.next_button.setText(_("Next"))
+        self.please_wait.setText(_("Please wait..."))
+
+    def select_and_save_language(self):
+        """Method for selecting and saving language in config file as {'language': <language-abbreviation: str>}"""
+        filtered_languages = dict(filter(lambda item: item[1] != 'Default', languages.items()))
```
Comment (cloudjrojek) [inline electrum/gui/qt/installwizard.py:203]: wouldn't it be more readable?
`{key: value for key, value in languages.items() if value != 'Default'}`
also, we are comparing translated 'Default' to plain 'Default'

File: electrum/gui/qt/installwizard.py [hunk 2]
```diff
@@ -690,7 +721,14 @@ def f(text):
...
 
-        warn_issue4566_label = WWLabel(MSG_PASSPHRASE_WARN_ISSUE4566)
+        warn_issue4566_label = WWLabel(
+            _("Warning") + ": " +
+            _("You have multiple consecutive whitespaces or leading/trailing "
+                "whitespaces in your passphrase.") + " " +
+            _("This is discouraged.") + " " +
+            _("Due to a bug, old versions of ELCASH Wallet will NOT be creating the "
+                "same wallet as newer versions or other software.")
+        )
```
Comment (cloudjrojek) [inline electrum/gui/qt/installwizard.py:731]: don't we have anything like lazy gettext available in app? i liked all consts on top much more :(

**Review by cloudjrojek — APPROVED** [review]

File: electrum/gui/qt/installwizard.py (hunk 2 above)
Comment (kj-crypto) [reply electrum/gui/qt/installwizard.py:731]: Unfortunately not

File: electrum/gui/qt/installwizard.py (hunk 1 above)
Comment (kj-crypto) [reply electrum/gui/qt/installwizard.py]: Done

**Review by rrozek — APPROVED** [review]

**Pull request merged**